/position/:ident -> /positon/id/:id

LudwikJaniuk opened the pull request:
Fixes #11 

Updated all the relevant places. New endpoint name so that bugs surface quickly in case I missed anything, or if anyone is still using the old identifier format.

LudwikJaniuk committed:
/position/:ident -> /positon/id/:id

Updates all the relevant places. New endpoint name so that bugs surface quickly in case I missed anything, or if anyone is still using the old identifier format.

JSimo reviewed (changes requested):
The old role-identifier api endpoints shouldn't be removed as it currently being used and is quite useful to have. (It is just that an id endpoint is nice to have as we can guarantee that it will never change, not that we want to remove the old one)

LudwikJaniuk committed:
/position/:ident -> /positon/id/:id

Updates all the relevant places. New endpoint name so that bugs surface quickly in case I missed anything, or if anyone is still using the old identifier format.

JSimo reviewed (changes requested):
Should have noticed it earlier, I thought the changes were made in the api endpoints. What needed changing to fix #11 was to add/edit api-endpoints not change the view urls, although it is probably a good change as well. I should have used an example url from the api instead. (I.e. http://localhost:5000/api/role/ordf instead of one being used for the views)

LudwikJaniuk commented:
No problem, då fixar jag det kanske imorgon

On Sat, Nov 5, 2016, 2:23 PM Jesper Simonsson notifications@github.com
wrote:

> _@JSimo_ requested changes on this pull request.
> 
> Should have noticed it earlier, I thought the changes were made in the api
> endpoints. What needed changing to fix #11
> https://github.com/datasektionen/dfunkt/issues/11 was to add/edit
> api-endpoints not change the view urls, although it is probably a good
> change as well. I should have used an example url from the api instead.
> (I.e. http://localhost:5000/api/role/ordf instead of one being used for
> the views)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/datasektionen/dfunkt/pull/13#pullrequestreview-7314139,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AGAbGVgIckB7sumV0zbkblkG_7Zqc493ks5q7INSgaJpZM4KoMZI
> .

LudwikJaniuk commented:
Ookej, api.js, de blir en lite större ändring, men yes, då är jag med på vad som ska göras

LudwikJaniuk committed:
Possible to get one role by id

LudwikJaniuk committed:
Possible to get role current mandates by id

LudwikJaniuk committed:
Clearer names for group identifiers

LudwikJaniuk commented:
@JSimo Nu ska det vara klart. Nya endpoints gjorda för de api-endpoints som använde role identifier, men de gamla är kvar, och jag ändrade namn från "identifier" till "groupIdentifier" där det var en sån som användes, så man inte blandar ihop dem. De där ändringarna jag gjorde med view endpoints har jag behållit.

Commit:
added id to /roles
